octobot_evaluators/api/evaluators.py

```diff
@@ -18,6 +18,7 @@
 from octobot_channels.channels.channel_instances import ChannelInstances
```

Comment:
I think we'll have to refactor this file after this PR. His methods become long and difficult to read maily due to a large number of params .